app/models/bishop.rb

```diff
@@ -1,2 +1,11 @@
 class Bishop < Piece
+
+  def display_piece
+    if black?
```

Comment:
Looks good!  These ifs are perfect candidates for inline ifs and relying on Ruby's implicit return of the last line of a method:

```ruby
def display_piece
  black? ? "&#9821;" : "&#9815;"
end
```